libs/sdk-ui-tests-e2e/README.md

```diff
@@ -6,37 +6,58 @@ This project contains End-to-End tests for GoodData.UI components.
 
 Tests in this library verify various go-through scenarios for GoodData.UI components. Tests are implemented using [Cypress](https://www.cypress.io/).
 There are two categories of tests: isolated and integrated. Isolated tests run against recorded backend responses, whereas integrated tests
-run against a live backend. Isolated tests run in the pre-merge phase and itegrated tests run in post-merge phase.
+run against a live backend. Isolated tests run in the pre-merge phase and integrated tests run in post-merge phase.
 
-## DEV Guide for isolated tests
+## Prepare env variables
 
-### Running tests locally
+```
+export TEST_BACKEND=https://abc.your-domain.com
+export TEST_BACKEND_NO_PREFIX=abc.your-domain.com
+export HOST=$TEST_BACKEND
+export IMAGE_ID=ui-sdk-scenarios
+
+```
+
+Provide all necessary info in the `.env` file. You can find more information in the file `.env.template`.
+
+For Tiger
 
-To run the tests against a live backend follow these steps:
+-   SDK_BACKEND=TIGER
+-   FIXTURE_TYPE=goodsales
+-   CYPRESS_TEST_TAGS=post-merge_integrated_tiger
+-   TIGER_API_TOKEN=
 
--   run `rush build -t sdk-ui-tests-e2e` to have the latest changes applied to the E2E tests package
--   run `npm run start-scenarios` to run the testing application. The application will run on port 9500
--   run `npm run cypress` to start cypress local instance
+For Bear
+
+-   SDK_BACKEND=BEAR
+-   FIXTURE_TYPE=goodsales
+-   CYPRESS_TEST_TAGS=post-merge_integrated_bear
+-   USER_NAME=bear@gooddata.com
+-   PASSWORD=
+-   AUTH_TOKEN=
+
+Add FILTER env variable with test file name to run and CYPRESS_TEST_TAGS env var to specify a comma-separated list of tags to run.
+
+To run the tests locally, you would need to build the sdk and the scenarios against which the tests run and then run the tests, i.e.
+
+## DEV Guide for isolated tests
+
+### Running tests locally
 
-The test application runs on `http://localhost:9500/gooddata-ui-sdk?scenario=<scenario-key>`, to show the Scenario you want to test, replace the `<scenario-key>` variable
-with the value of respective Scenario key. You can find all available keys in the `ComponentResolver.tsx`.
+To run the tests against a mock backend follow these steps:
 
-To log in to the scenario app, visit `http://localhost:9500/gdc/account/login`
+-   Run `rush build -t sdk-ui-tests-e2e` to have the latest changes applied to the E2E tests package.
```

Comment:
it seems like that there is a command to run isolated locally in package.json. Please try to this command if it works **run-isolated-local**, then we should replace these lines by this command

Reply:
it just rebuild rush for sdk-ui-tests-e2e, it is redundant so I will remove it.

Reply:
Please try to this command if it works run-isolated-local, then we should replace these lines by this command